README.md

```diff
@@ -28,6 +29,11 @@ Download the latest binary executable for your operating system:
   sudo tar xvzf tkn_0.4.0_Darwin_x86_64.tar.gz -C /usr/local/bin tkn
   ```
 
+* Windows
+
+  - Uncompress the [zip file](https://github.com/tektoncd/cli/releases/download/v0.4.0/tkn_0.4.0_Windows_x86_64.zip)
+    and use it from the Windows PowerShell.
```

Comment:
I don't have a windows workstation so I can't really test those instructions, i had it vague enough because I assume that someone that goes to the relative *pain* to use tekton on windows would know enough about command line to take care of her/his/self.

I don't think the goal here is to give a windows command line tutorial,

nice, nice for choco but we need some windows users who wants to handle it first,